services/management/checks.go

```diff
@@ -212,24 +212,60 @@ func (s *ChecksAPIService) ListSecurityChecks(ctx context.Context, req *manageme
 		m[c] = struct{}{}
 	}
 
-	checks, err := s.checksService.GetChecks()
+	results, err := s.checksService.GetChecks()
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get available checks list")
 	}
 
-	res := make([]*managementpb.SecurityCheck, 0, len(checks))
-	for _, c := range checks {
-		_, disabled := m[c.Name]
-		res = append(res, &managementpb.SecurityCheck{
-			Name:        c.Name,
-			Disabled:    disabled,
-			Summary:     c.Summary,
-			Description: c.Description,
-			Interval:    convertInterval(c.Interval),
-		})
+	checks := make([]*managementpb.SecurityCheck, 0, len(results))
+	if req != nil && req.FilterParams != nil {
+		if req.FilterParams.Category != nil {
```

Comment:
in case of FilterParams passed, but categories isn't passed in request we will return empty result. I'm not sure if it's a good idea.